### ISSUE TITLE
Fix Batch message count update event when selecting all (Safari only)

### ISSUE THREAD

**Issue by sethshoultes** [eventespresso/event-espresso-core#729]
**Note: Steps to replicate:
Safari browser, click the select all checkbox to select all registrations. The count stays at zero. Other browsers work as expected.**

@garthkoyle commented on [Fri May 25 2018](https://github.com/eventespresso/eventsmart.com-website/issues/28)

I'm pretty sure we've fixed this before but I can't find the Codebase ticket. This doesn't happen on Event Smart.

On MENW, when you select the number of Batch Messages to send, the count is incorrect. I can select the entire list by clicking the Bulk action, but the count stays 0. Then, when I **de-select** the bulk option, the count then increases.

So, go to Registrations > bulk select the registrations > notice that the "Send Batch Message" count is incorrect.

Using Safari on OSX.
![image](https://user-images.githubusercontent.com/3094547/40556567-5f3d139c-600a-11e8-917a-ac4cbeb7d809.png)

---

@garthkoyle commented on [Mon Oct 15 2018](https://github.com/eventespresso/eventsmart.com-website/issues/28#issuecomment-429867810)

This is still a bug.

---

@sethshoultes commented on [Mon Oct 15 2018](https://github.com/eventespresso/eventsmart.com-website/issues/28#issuecomment-429896558)

I can't seem to reproduce this issue. Here's are some videos from Event Smart:
Safari: http://take.ms/W7rBQ
Google Chrome: https://monosnap.com/file/zYI4pOFIO4yFnl8XFSvcknxCXA9dKr#
Firefox: https://monosnap.com/file/nRmb9Wb0EqlC8IeOBRc1qK1Lr6lMvC#

Here are some videos from MENW:
Safari on Mac: https://monosnap.com/file/7hXN0jRpWLCIDUfpC6EVNIRvxkrJMm#
Google Chrome: https://monosnap.com/file/JizPk5S2B6gCpaM6fV3m8iu5xCGIEX#
Firefox: https://monosnap.com/file/4dbvcJSaBujsqhC0jsrT69DMMbCMJm#

However, while testing, I did note that when clicking on a registration record, then click the back button, the selections become unselected.

Do you have any more details on how to reproduce this issue? Or maybe a site where we can see this issue in action?

---

@garthkoyle commented on [Mon Oct 15 2018](https://github.com/eventespresso/eventsmart.com-website/issues/28#issuecomment-429898230)

Try it again with the "bulk" action, not individual messages. Here is another video: https://www.screencast.com/t/6IP9hCl4bObU

---

@sethshoultes commented on [Mon Oct 15 2018](https://github.com/eventespresso/eventsmart.com-website/issues/28#issuecomment-429903489)

Okay, I see it now. It looks like this is an issue with EE4 core, so I think we should move this ticket to that repo.

**Comment by sethshoultes** [comment]
This was originally reported on Event Smart: https://github.com/eventespresso/eventsmart.com-website/issues/28

**Comment by joshfeck** [comment]
original codebase ticket:
https://events.codebasehq.com/projects/event-espresso/tickets/10446

**Comment by joshfeck** [comment]
This is working on self-hosted EE4.

![screen shot 2019-02-13 at 5 28 11 pm](https://user-images.githubusercontent.com/891156/52748570-dbd07000-2fb4-11e9-8ae6-6d664f992940.png)

**Comment by nerrad** [comment]
I'm unable to reproduce this in Safari 12.1 (build 4607.1.40.1.4) using the steps outlined in the original post:

> Note: Steps to replicate:
Safari browser, click the select all checkbox to select all registrations. The count stays at zero. Other browsers work as expected.

![screenshot of attempt](https://cl.ly/5b17d576de50/Screen%20Recording%202019-04-25%20at%2008.37%20PM.gif)

I wonder if this was a bug in Safari that got fixed in a later version or maybe we fixed the bug at some point since the original report?
